### DISCUSSION TITLE
What are the 7 Wonders of the Ancient World?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#830] [Galaxy Trivia] [answered]
**Bonus:** Add the country emoji they are in.

**Comment by peterbaccaro** [comment]
The Seven Wonders of the Ancient World were a collection of remarkable architectural and artistic achievements of the ancient world. Although most of them no longer exist today, they continue to inspire awe and curiosity. The original Seven Wonders of the Ancient World are:

1. 🇪🇬 **The Great Pyramid of Giza**: Located in **Egypt**, the Great Pyramid of Giza is the only wonder that still partially stands. Built as a tomb for the pharaoh Khufu, it is the largest and oldest of the three pyramids in the Giza pyramid complex.

2. 🇮🇶 **Hanging Gardens of Babylon**: Believed to have been built in present-day **Iraq**, the Hanging Gardens were a terraced garden with a variety of plants and trees. They were said to have been constructed by King Nebuchadnezzar II for his wife, who missed the green landscapes of her homeland.

3. 🇬🇷 **Statue of Zeus at Olympia**: Located in **Greece**, the Statue of Zeus was a massive seated statue of the Greek god Zeus. Created by the renowned sculptor Phidias, it stood within the Temple of Zeus in Olympia and was considered one of the greatest statues of the ancient world.

4. 🇹🇷 **Temple of Artemis at Ephesus**: Situated in present-day **Turkey**, the Temple of Artemis was a grand Greek temple dedicated to the goddess Artemis. It was renowned for its impressive size and richly decorated architecture.

5. 🇹🇷 **Mausoleum at Halicarnassus**: The Mausoleum was a monumental tomb built for King Mausolus of Caria, located in present-day **Turkey**. It was a grand structure adorned with intricate sculptures and friezes, considered a marvel of ancient architecture.

6. 🇬🇷 **Colossus of Rhodes**: The Colossus of Rhodes was a gigantic bronze statue of the Greek god Helios, erected on the **island of Rhodes in Greece**. It was one of the tallest statues of the ancient world, standing at the entrance of the harbour.

8. 🇪🇬 **Lighthouse of Alexandria**: Situated in **Egypt**, the Lighthouse of Alexandria was a towering structure built on the island of Pharos. It served as a navigational aid for ships entering the port of Alexandria and was considered one of the tallest man-made structures of its time.

It is important to note that, apart from the Great Pyramid of Giza, all of the original Seven Wonders have been destroyed or lost over the centuries. Nevertheless, their historical significance and architectural achievements continue to captivate our imagination.

**Comment by SANKARAMDAS** [comment, marked as answer]
Amid the march of progress throughout the modern world, evidence of Earth’s rich past exists in historical constructions that tell stories of human achievement. Among these, seven were deemed “wonders”: hallowed examples of the contributions of ancient Mediterranean and Middle Eastern civilizations.

* 🇪🇬 The great Pyramid of Giza in Egypt - stood as the world’s tallest human-made structure for nearly 4,000 years. Erected around 2560 B.C.E.

* 🇮🇶 Hanging Gardens of Babylon in Iraq - were believed to have been built around 600 B.C.E., by Nebuchadnezzar II, who ruled Babylonia from 605–562 B.C.E.

* 🇬🇷 Statue of Zeus in Greece - Many of those who gazed upon the 12-meter (40-foot) statue commented that the proportions of Zeus inside the temple were off. Regardless, the statue was revered by most as a stunning earthly representation of the great god that filled passersby with awe—until it was destroyed in an earthquake in the fifth century B.C.E.

* 🇹🇷 Temple of Artemis in Turkey - The most famous iteration, completed in 550 B.C.E., was about 115 meters (377 feet) long, 55 meters (200 feet) wide, included 127 Ionic-styled columns, and was decorated with fine sculptures and paintings.'

* 🇹🇷 Mausoleum of Halicarnassus in Turkey -The mausoleum was designed between 353 and 350 B.C.E. by the Greek architects Satyrus and Pythis.

* 🇬🇷 Colossus of Rhodes in Greece -Erected between 292 and 280 B.C.E. by the sculptor Chares of Lindos, and standing at about 33 meters (105-110 feet), the statue was a victory monument designed to honor the defeat of the invading army of Demetrius in 304 B.C.E.

* 🇪🇬 Pharos (Lighthouse) of Alexandria in Egypt -  It was completed between 285 and 247 B.C.E.

https://education.nationalgeographic.org/resource/seven-wonders-ancient-world/

**Comment by anjutares** [comment]
The Seven Wonders of the Ancient World are:

    The Great Pyramid of Giza 🇪🇬 (Egypt)
    Hanging Gardens of Babylon 🇮🇶 (Iraq)
    Statue of Zeus at Olympia 🇬🇷 (Greece)
    Temple of Artemis at Ephesus 🇹🇷 (Turkey)
    Mausoleum at Halicarnassus 🇹🇷 (Turkey)
    Colossus of Rhodes 🇬🇷 (Greece)
    Lighthouse of Alexandria 🇪🇬 (Egypt)

Please note that only the Great Pyramid of Giza still exists today, while the rest have been destroyed or lost to history.
